crates/sui-indexer-alt-graphql/src/api/types/transaction_kind/change_epoch.rs

```diff
@@ -19,11 +28,12 @@ pub(crate) struct ChangeEpochTransaction {
     pub(crate) native: NativeChangeEpoch,
 }
 
-// TODO(DVX-1158): Support systemPackages.
+pub(crate) type CSystemPackage = JsonCursor<usize>;
+
 /// A system transaction that updates epoch information on-chain (increments the current epoch). Executed by the system once per epoch, without using gas. Epoch change transactions cannot be submitted by users, because validators will refuse to sign them.
 ///
 /// This transaction kind is deprecated in favour of `EndOfEpochTransaction`.
-#[Object]
+#[GraphQLObject]
```

Comment:
nit: Leave this as `Object` (the framework's `Object` and our `Object` type do not conflict here, because they are in different namespaces, attributes vs values).